cfme/v2v/conversion_hosts.py

```diff
@@ -0,0 +1,240 @@
+                               "osp_ca_cert": self.osp_ca_cert}
+        }
+
+    @property
+    def is_host_configured(self):
+        """Wait for conversion host to configure and return success/failure state"""
+        view = navigate_to(self.parent, "All")
+        try:
+            wait_for(
+                lambda: not view.conversion_host_progress.in_progress(self.hostname),
+                message="Conversion Host configuration is in progress",
+                delay=5,
+                num_sec=300
+            )
+        except TimedOutError:
+            self.logger.warning("Timed out waiting for {} to configure".format(self.hostname))
+        return view.conversion_host_progress.is_host_configured(self.hostname)
+
+
+@attr.s
+class ConversionHostCollection(BaseCollection):
+    """Collection object for conversion host object"""
+
+    ENTITY = ConversionHost
+
+    def create(self,
+               target_provider,
+               cluster,
+               hostname,
+               conv_host_key,
+               transformation_method,
+               vddk_library_path=None,
+               vmware_ssh_key=None,
+               osp_cert_switch=None,
+               osp_ca_cert=None):
+        """Configure new conversions host in UI
+        Args:
+            target_provider: RHV or OSP
+            cluster: Default
+            hostname: Hostname to be configured
+            conv_host_key: /etc/pki/ovirt-engine/keys/engine_id_rsa on RHV-M.
+            transformation_method: vddk or ssh
+            vddk_library_path: url for vddk
+            vmware_ssh_key: /var/lib/vdsm/.ssh/id_rsa on rhv host
+            osp_cert_switch: Yes or No
+            osp_ca_cert: Tls Cert
+
+        """
+        conversion_host = self.instantiate(
+            target_provider=target_provider,
+            cluster=cluster,
+            hostname=hostname,
+            conv_host_key=conv_host_key,
+            transformation_method=transformation_method,
+            vddk_library_path=vddk_library_path,
+            vmware_ssh_key=vmware_ssh_key,
+            osp_cert_switch=osp_cert_switch,
+            osp_ca_cert=osp_ca_cert
+        )
+
+        view = navigate_to(self, "Configure")
+        view.fill(conversion_host.fill_dict())
+        return conversion_host
+
+
+@navigator.register(ConversionHostCollection, "All")
+class ConversionHosts(CFMENavigateStep):
+    prerequisite = NavigateToAttribute("appliance.server", "LoggedIn")
+    VIEW = MigrationSettingsView
+
+    def step(self):
+        self.prerequisite_view.navigation.select("Compute", "Migration", "Migration Settings")
+        self.view.ConversionHostsTab.click()
+        self.view.wait_displayed('20s')
+
+
+@navigator.register(ConversionHostCollection, "Configure")
+class ConfigureConversionHost(CFMENavigateStep):
+    VIEW = ConfigureConversionHostsView
+    prerequisite = NavigateToSibling("All")
+
+    def step(self):
+        self.prerequisite_view.wait_displayed('20s')
```

Comment:
Same thing here - NavigateToSibling is going to be using wait_for_view default of 10s, I think this wait is redundant.

If you want to wait for `self.prerequisite_view.configure_conversion_host.wait_displayed`, before clicking, that would be more appropriate here.